### PR TITLE
Mobility auto

### PR DESCRIPTION
**Pull request by Gavin-Niederman**
make the robot actually drive straight

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.kt [hunk 1]
```diff
@@ -45,7 +45,7 @@ object RobotContainer {
         setDefaultCommands()
         DriverStation.silenceJoystickConnectionWarning(RobotBase.isSimulation())
 
-        LimelightHelpers.setCameraMode_Driver("limelight")
+        LimelightHelpers.setCameraMode_Driver("limelight");
```
Comment (lewisfm) [inline src/main/java/frc/robot/RobotContainer.kt]: whats this bruh

Comment (Gavin-Niederman) [reply src/main/java/frc/robot/RobotContainer.kt]: Idk its just there

**Review by lewisfm — APPROVED** [review]
LGTM (Let's Generate Technical Milestones)

**Review by rotatingcow — APPROVED** [review]
lemonade glows through midnight

**Pull request merged**